packages/core/src/html/NodeProcessor.js

```diff
@@ -194,6 +194,7 @@ class NodeProcessor {
       case 'style':
         processScriptAndStyleTag(node, this.userScriptsAndStyles);
         break;
+      case 'disqus':
```

Comment:
Thanks! How about just renaming the tag? So this fix lives strictly in the plugin

Reply:
@ang-zeyu Thanks for the suggestion! We should really avoid unnecessary coupling. I've made the changes to avoid the coupling. 

Regarding "renaming the tag", I'm not sure if we are thinking of the same thing. Let me know if you actually had another idea in mind!